lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1450,13 +1450,21 @@ void ApplyWorkingDir(SmallString &clang_argument, StringRef cur_working_dir) {
   llvm::sys::path::remove_dots(clang_argument);
 }
 
-std::array<StringRef, 2> macro_flags = { "-D", "-U" };
+#define MACRO_FLAGS "-D", "-U"
+#define MULTI_ARG_UNIQUE_FLAGS MACRO_FLAGS, "-I", "-F"
```

Comment:
Let's not use macros for this. The list is so short that duplicating -D and -U is not going to be a big deal

Reply:
We *could* include the `Options.inc` Clang header file generated by tablegen (or `Options.h`) that to define an enum and string values for the options to avoid duplication.

Reply:
The problem is not in the type of the values, but in missing way to concatenate two compile-time array constants. How would generated code help here?